.pre-commit-config.yaml

```diff
@@ -13,3 +13,11 @@ repos:
       - id: black
         args:
         - --preview
+
+  - repo: https://github.com/pycqa/flake8
+    rev: 6.1.0
+    hooks:
+    - id: flake8
+      args: [--append-config=.flake8-pre-commit.cfg]
+      additional_dependencies: [attrs>=19.2.0]
```

Comment:
Why don't we need flake8 here? Can't our pyproject.toml handle this to avoid copy pasta / more things to change in the future?